tests/product/test_package_install.py

```diff
@@ -91,91 +91,6 @@ def test_install_exclude_nodes(self):
         for container in self.cluster.all_hosts():
             self.installer.assert_uninstalled(container, msg=output)
 
-    def test_install_invalid_path(self):
-        rpm_name = self.installer.copy_presto_rpm_to_master()
-        cmd_output = self.run_prestoadmin('package install /mnt/presto-admin'
-                                          '/invalid-path/presto.rpm',
-                                          rpm=rpm_name, raise_error=False)
-        error = '\nFatal error: [%s] error: ' \
-                '/mnt/presto-admin/invalid-path/presto.rpm: open failed: ' \
-                'No such file or directory\n\nAborting.\n'
-        expected = ''
-        for host in self.cluster.all_internal_hosts():
-            expected += error % host
-
-        self.assertEqualIgnoringOrder(cmd_output, expected)
-
-    def test_install_no_path_arg(self):
```

Comment:
The fact that those tests are using a separate command doesn't matter? I assume the same logic is exercised?